packages/story-editor/src/components/form/color/colorInput.js

```diff
@@ -224,17 +224,15 @@ const ColorInput = forwardRef(function ColorInput(
         placement={PLACEMENT.LEFT_START}
         spacing={spacing}
         invisible={isEyedropperActive}
-        renderContents={({ propagateDimensionChange }) => (
```

Comment:
Hm, I think the propagation might still be needed, for example when going from view 2 to the Default view, the lower part can get hidden otherwise:
![propagation](https://user-images.githubusercontent.com/3294597/129170414-022a360b-d274-4698-a01a-71b8194ac21d.gif)

Reply:
Q: Previously, the Saved colors had a scrollbar if more then 3 lines, should it be the same now, too?
![Screenshot 2021-08-12 at 12 28 27](https://user-images.githubusercontent.com/3294597/129173860-849b9b76-dca2-4540-b9da-6f0abb27ee0b.png)

Reply:
> Hm, I think the propagation might still be needed, for example when going from view 2 to the Default view, the lower part can get hidden otherwise:

Dammit, I hoped you wouldn't notice! 😄 

I did see this one time but forgot again. I will file as a follow-up bug, as it's pretty low priority.

> Q: Previously, the Saved colors had a scrollbar if more then 3 lines, should it be the same now, too?

Will be fixed in #8656 (only two lines to be shown).

Reply:
> > Hm, I think the propagation might still be needed, for example when going from view 2 to the Default view, the lower part can get hidden otherwise:
> 
> I will file as a follow-up bug, as it's pretty low priority.

Filed in #8673

Reply:
Looks like it will be handled in #8656 (via 3 dots button, max 2 rows)
![image](https://user-images.githubusercontent.com/21036224/129276592-0a026107-b332-408d-aefd-896c597a305d.png)